source/features/conversation-activity-filter.tsx

```diff
@@ -20,9 +21,9 @@ const states = {
 
 type State = keyof typeof states;
 
-let currentSetting: State = 'default';
```

Comment:
If we're dropping the global "tracking" variable, let's rename the local instances to just `setting`. Also `applyCurrentSetting()` → `applySetting(setting)`.

Reply:
I've planned to rename the variables, in a separate commit to keep the noise out.